packages/expo-yarn-workspaces/index.js

```diff
@@ -62,11 +69,9 @@ exports.createMetroConfiguration = function createMetroConfiguration(projectPath
     },
 
     transformer: {
+      ...defaultConfig.transformer,
       // Ignore file-relative Babel configurations and apply only the project's
       enableBabelRCLookup: false,
-
-      // Temporarily include the Expo asset plugin; figure out a more general way to include it
-      assetPlugins: ['expo/tools/hashAssetFiles'],
```

Comment:
What replaces this?

Reply:
It's comes from `@expo/metro-config`
https://github.com/expo/expo-cli/blob/master/packages/metro-config/src/ExpoMetroConfig.ts#L34-L42

https://github.com/expo/expo-cli/blob/master/packages/metro-config/src/ExpoMetroConfig.ts#L94

Reply:
`@expo/metro-config` https://github.com/expo/expo-cli/blob/610c5c60413a2cae824c7b4d25f41244cf71eef8/packages/metro-config/src/ExpoMetroConfig.ts#L36